notify/impl.go

```diff
@@ -198,7 +198,7 @@ func (w *Webhook) Notify(ctx context.Context, alerts ...*types.Alert) error {
 	resp.Body.Close()
 
 	if resp.StatusCode/100 != 2 {
-		return fmt.Errorf("unexpected status code %v", resp.StatusCode)
+		return fmt.Errorf("Webhook: unexpected status code %v from %s", resp.StatusCode, w.URL)
```

Comment:
The integration names should be provided via structured logging `log.With("integration", "webhook").X(err)` on the caller side.